docs/section10/10-1.md

```diff
@@ -0,0 +1,41 @@
+# Airside systems – Rule 10-1  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** False    
+**Rule ID:** 10-1  
+ 
+**Rule Description:** The proposed design includes humidification (which does not comply with 90.1-2019 Section 6.5.2.4) and the baseline building design has been modeled with adiabatic humidification, as expected.  
+
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:** 
+
+1. The zone in the P-RMR includes humidification.
+
+**Function Call:** 
+
+1. match_data_element()
+2. get_zones_hvac_system_humid_p()
+3. get_list_hvac_systems_associated_with_zone()
+
+**Manual Check:** Check that the applicable baseline HVAC systems were modeled per the rules of G3.1-10 HVAC System baseline column which states that if the proposed design includes humidification then the baseline building design shall use adiabatic humidification (if the proposed humidification system does not comply with 90.1-2019 Section 6.5.2.4). For each applicable zone (i.e. zones with humidification in the proposed) check if the humidification system associated with the zone complies with 90.1-2019 Section 6.5.2.4. If it does NOT then check that the baseline system serving the zone was modeled with adiabatic humidification. 
+ 
+**Rule Logic:**  
+**Applicability Check 1:** 
+- Call function to get dictionary with applicable zones (key) and hvac systems (values) in P_RMR: `applicable_zones_hvac_sys_humid_dict_p = get_zones_hvac_system_humid_p(P_RMR)`
+- For each zone in P_RMR: `zone_p in P_RMR...Zone:`
+    - Check if the zone has humidification in the P_RMR, if so then proceed with logic: `if applicable_zones_hvac_sys_humid_dict_p.has_key(zone_p.id):`
+        - Get analogous zone ID in the B_RMR: `zone_b = match_data_element(B_RMR,Zones,zone_p.id)`
+        - Get list of hvac systems associated with the zone (should only be one): `heating_ventilation_air_conditioning_systems_b = get_list_hvac_systems_associated_with_zone(B_RMR, zone_b.id)`
+        - Add to list of heating_ventilation_air_conditioning_systems_list_b as the code loops through the zones: `heating_ventilation_air_conditioning_systems_list_b = heating_ventilation_air_conditioning_systems_list_b.append(heating_ventilation_air_conditioning_systems_b)`                
+- Convert the list of heating_ventilation_air_conditioning_systems_list_b associated with the applicable zones to a set and the back to a list to eliminate duplicates after looping through all zones: `heating_ventilation_air_conditioning_systems_list_b = list(set(heating_ventilation_air_conditioning_systems_list_b))`
```

Comment:
For the applicability check, I would do the following:
1. Cycle through all HVAC systems in the proposed design to see if any of them have heatingsystem.humidity_type != None OR Null? If yes, the RDS applies and we need to identify the corresponding baseline systems as described in the following steps. 
2. Identify zones served by these proposed systems (I think we already have a function for that?), and then use another (I think existing) function to identify HVAC systems in the baseline that serve at least one of these zones.  HVAC systems in the baseline that serve